web/src/components/buttons/DownloadCsv.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button } from 'components/Button';
+import { Download } from 'lucide-react';
+import { useParams } from 'react-router-dom';
+import { DEFAULT_ICON_SIZE } from 'utils/constants';
+
+import { getCountryName, getZoneName } from '../../translation/translation';
+
+function getCountryToDownload(zoneId?: string) {
+  if (!zoneId) {
+    return '';
+  }
+
+  const zoneName = getZoneName(zoneId);
+  const zoneCountryName =
+    zoneId && zoneId.includes('-') && getCountryName(zoneId)?.toLowerCase();
+  return zoneCountryName || zoneName;
+}
+
+export function DownloadCsv() {
+  const { zoneId } = useParams();
+  const countryToDownload = getCountryToDownload(zoneId);
+
+  const url = `https://www.electricitymaps.com/data-portal/${countryToDownload.toLowerCase()}?utm_source=app&utm_medium=download_button&utm_campaign=csv_download`;
+
+  return (
+    <Button
+      href={url}
+      icon={<Download size={DEFAULT_ICON_SIZE} />}
+      type="tertiary"
+      size="md"
+      aria-label="Download CSV Data"
+    />
```

Comment:
Can we mark that this is an external link just like we do in the header the desktop view?

Reply:
In line with trying to unify the journey I think this should feel as "one platform" as possible, I feel the external link will feel disconnected from the unified experience

Reply:
I get that but this will also be available on mobile which could lead to unexpected navigations.

I don't think that should be a blocker though.

Reply:
Looks like the new portal does the same thing: ![image](https://github.com/user-attachments/assets/0f3e947a-ddbd-48f9-8220-5c8f9ec6e77c)